screenpy_selenium/resolutions/custom_matchers/is_invisible_element.py

```diff
@@ -0,0 +1,46 @@
+"""
+A matcher that matches an invisible element. For example:
+
+    assert_that(driver.find_element_by_id("search"), is_invisible_element())
+"""
+
+
+from typing import Optional
+
+from hamcrest.core.base_matcher import BaseMatcher
+from hamcrest.core.description import Description
+from selenium.webdriver.remote.webelement import WebElement
+
+
+class IsInvisibleElement(BaseMatcher[Optional[WebElement]]):
```

Comment:
Is this better/more correct than doing `IsNot(Visible())`?

Reply:
IMO, It’s preference. 

Mostly in readable language and logging.  I think this aligns  with expected conditions. Someone looking for similar behavior would find `IsInvisible` because they were used to `EC.invisibility_of_element_located`. 

I don’t see anything wrong with being able to write the behavior either way.

Kinda like `not True` vs `False`. Both work fine, the choice to use either comes down to style.